### PR TITLE
devenv: reintroduce sbtc-bridge-api

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
## Description
This PR fixes the sbtc-bridge-api in devnet. It also requires the use of mongodb which has been added as well.

## Checklist
- [x] Additions and modifications are documented
- [x] Additions and modifications are tested

re: https://github.com/stacks-network/sbtc/issues/141

### CODE REVIEW

File: devenv/docker-compose.yml [hunk 1]
```diff
@@ -166,6 +166,42 @@ services:
...
+      btcSchnorrReveal: changeme
+      btcSchnorrReclaim: changeme
+      btcNode: bitcoin:18443
+      btcRpcUser: devnet
+      btcRpcPwd: devnet
+      network: testnet
+      bitcoinExplorerUrl: http://bitcoin-explorer:3002/api
+      stacksExplorerUrl: http://stacks-explorer:3000
+      stacksApi: http://stacks-api:3999
+      sbtcContractId: ST1R1061ZT6KPJXQ7PAXPFB6ZAZ6ZWW28G8HXK9G5.related-pink-tick
```
[the remark this answers is not in the document]
Comment (friedger) [reply devenv/docker-compose.yml:200]: Yes: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.asset

File: devenv/docker-compose.yml [hunk 2]
```diff
@@ -166,6 +166,42 @@ services:
...
+      - stacks-explorer
+      - bitcoin-explorer
+      - mongodb
+      - sbtc
+    ports:
+      - 3010:3010
+    environment:
+      NODE_ENV: dev
+      btcSchnorrReveal: changeme
+      btcSchnorrReclaim: changeme
```
[the remark this answers is not in the document]
Comment (friedger) [reply devenv/docker-compose.yml:192]: What would go there? The credentials are all known via the mnemonic

**Review by CAGS295 — APPROVED** [review]

File: devenv/docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (radicleart) [reply devenv/docker-compose.yml:192]: they were for testing and can be removed.. i'll do this asap.

**Review by friedger — APPROVED** [review]
LGTM

**Pull request merged**